README.md

```diff
@@ -56,8 +56,9 @@ Alex Chernyak ([@alexme22](https://github.com/alexme22)),
 Dillon George ([@dillonrg](https://github.com/dillonrg)),
 Sean Lawlor ([@slawlor](https://github.com/slawlor)),
 Kevin Lewi ([@kevinlewi](https://github.com/kevinlewi)),
-Jasleen Malvai ([@jasleen1](https://github.com/jasleen1)), and
-Ercan Ozturk ([@eozturk1](https://github.com/eozturk1)).
+Jasleen Malvai ([@jasleen1](https://github.com/jasleen1)),
+Ercan Ozturk ([@eozturk1](https://github.com/eozturk1)), and
+Hao Chen ([@haochenuw](https://github.com/haochenuw)).
```

Comment:
Let's revert this for now, this is the list of "original authors" and we can update this section to give you attribution at some point later, but at the very least in a separate PR.

Reply:
sounds good, reverted!